### ISSUE TITLE
[TLP]: 

### ISSUE THREAD

**Issue by LowMcCarey** [soonaverse/soonaverse-dao#850]
### Email

lowmccarey@web.de

### Company Name

Rush Token Support

### Your Position

CEO

### Name

LowMcCarey

### Have you already done a public sale?

Yes

### Files and other information

```shell
Token
https://soonaverse.com/space/0x41192a2d3a8d01c02994cb05d557f817a319570c/overview

Talk:
https://discord.com/invite/y5eFej49

Stream Channel:
https://www.twitch.tv/rush_runner
```

**Comment by shonuffy** [comment]
Please apply to the KYC applicant award, so you can gain access to the TLP KYC NFT, which you will have to purchase. Once that is done I will forward your information to our KYC vendor who will contact you directly.

**Comment by LowMcCarey** [comment]
> Please apply to the KYC applicant award, so you can gain access to the TLP KYC NFT, which you will have to purchase. Once that is done I will forward your information to our KYC vendor who will contact you directly.

I need help again please. Where can I register for the KYC procedure?

**Comment by shonuffy** [comment]
https://discord.com/channels/892481940953915433/947920161195851797/994886573004959785

**Comment by LowMcCarey** [comment]
> https://discord.com/channels/892481940953915433/947920161195851797/994886573004959785

Sorry, but the Discord link isn't working.
Discord says either you don't have access or this channel doesn't exist.

My Discord Name: Low_McCarey#6760

**Comment by shonuffy** [comment]
Applicant award - https://soonaverse.com/award/0x4d02b346167cca51bec4a20576dd4c75a5a35ec9/overview

KYC NFT - https://soonaverse.com/collection/0x779f686073adfd2c2962e1d4d749f13e179f8e0b

**Comment by shonuffy** [comment]
@LowMcCarey Please advise if this is something you want to continue to pursue.

**Comment by LowMcCarey** [comment]
Yes, but I don't get any IOTAs on my IOTA EVM wallet to buy the launch pad token.

**Comment by shonuffy** [comment]
MM is just for authentication, you use Firefly to but the NFT

**Comment by LowMcCarey** [comment]
I don't get an option for the Firefly. It just aborts when trying to buy and says there is not enough in the wallet.

**Comment by shonuffy** [comment]
I've approved your badge, please complete the NFT purchase

**Comment by LowMcCarey** [comment]
Ok thx. It works :)

**Comment by shonuffy** [comment]
Please let me know if you are continuing through the process.

**Comment by LowMcCarey** [comment]
I have completed the NFT purchase. What would be the next step now? :)

**Comment by shonuffy** [comment]
Our KYC vendor will be contacting you via email.

**Comment by LowMcCarey** [comment]
How long do I have to wait for contact now? :)

**Comment by shonuffy** [comment]
I sent a reminder to the vendor, will try to have this wrapped up quickly. Thank you for your patience.

**Comment by LowMcCarey** [comment]
Can you contact me when it's ready? Because I haven't received any confirmation yet. Although I had my KYC deposited as requested per the instructions right after they contacted me via email. But I want to make sure I don't miss anything, for example if future emails end up in the spam folder. 

Thank you again :)

**Comment by shonuffy** [comment]
Yup, you are good. Can you send me a link to the configured token?

**Comment by LowMcCarey** [comment]
Hello again,

my token is the following:
https://soonaverse.com/token/0xbe507db0f565c4b7a145f32daaed6bd6655fda7c/metrics

**Comment by shonuffy** [comment]
Your token is live. Good luck.

**Comment by LowMcCarey** [comment]
Ok, thx 😊

Gesendet von Mail für Windows

Von: shonuffy
Gesendet: Montag, 17. Oktober 2022 04:54
An: soonaverse/soonaverse-dao
Cc: LowMcCarey; Mention
Betreff: Re: [soonaverse/soonaverse-dao] [TLP]: (Issue #850)

Your token is live. Good luck.
